ddtrace/profiling/collector/stack.pyx

```diff
@@ -247,16 +259,35 @@ cdef get_thread_name(thread_id):
             return "Anonymous Thread %d" % thread_id
 
 
-cdef stack_collect(ignore_profiler, thread_time, max_nframes, interval, wall_time, thread_span_links):
-    current_exceptions = []
+cpdef get_thread_native_id(thread_id):
+    try:
+        thread_obj = threading._active[thread_id]
+    except KeyError:
+        # This should not happen, unless somebody started a thread without
+        # using the `threading` module; in that case, ignore the thread altogether.
```

Comment:
```suggestion
        # using the `threading` module
```
I think the line below is the more accurate statement?

Reply:
Yes you're right, I might have changed my mind at some point :)